libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -453,12 +455,41 @@ public void appendMediaFile(final MediaFile mediaFile, final String mediaUrl, Im
                 // TODO: insert video
                 ToastUtils.showToast(getActivity(), R.string.media_insert_unimplemented);
             } else {
-                // TODO: insert image
-                ToastUtils.showToast(getActivity(), R.string.media_insert_unimplemented);
+                imageLoader.get(mediaUrl, new ImageLoader.ImageListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
```

Comment:
Good idea.  I added a message and failed icon on error response with f40bf8a.

As for the `downloadedBitmap == null` condition in `onResponse`, adding an error message and/or icon for the `downloadedBitmap == null` case would be confusing when the load was successful since `onResponse` is called twice with at least one instance of `downloadedBitmap == null` regardless of success/fail.